geniza/corpus/tests/test_corpus_templates.py

```diff
@@ -55,7 +55,7 @@ def test_viewer(self, client, document):
         response = client.get(document.get_absolute_url())
         assertContains(
             response,
-            '<section id="itt-panel">',
+            '<section id="itt-panel" data-controller="iiif">',
```

Comment:
why is the controller at this level? shouldn't each deep zoom image be bound to its own instance of the controller?